packages/kbn-managed-content-badge/index.ts

```diff
@@ -9,9 +9,10 @@ import { i18n } from '@kbn/i18n';
 import type { EuiToolTipProps } from '@elastic/eui';
 import type { TopNavMenuBadgeProps } from '@kbn/navigation-plugin/public';
 
-export const getManagedContentBadge: (tooltipText: string) => TopNavMenuBadgeProps = (
-  tooltipText
-) => ({
+export const getManagedContentBadge: (
+  tooltipText: string,
+  disableTooltipProps: boolean
+) => TopNavMenuBadgeProps = (tooltipText, disableTooltipProps = false) => ({
```

Comment:
nit: can we flip the disable flag to an `enable` one?

```suggestion
) => TopNavMenuBadgeProps = (tooltipText, enableTooltipProps = true) => ({
```

I find reason with positive flags to be easier to read compared to negative ones (and double negative ones!).